src/modules/blockchainApplication/explore/components/BlockchainApplicationRow/BlockchainApplicationRow.js

```diff
@@ -1,12 +1,15 @@
-import React, { useCallback } from 'react';
+import React from 'react';
 import grid from 'flexboxgrid/dist/flexboxgrid.css';
 import TokenAmount from '@token/fungible/components/tokenAmount';
 import { usePinBlockchainApplication } from '@blockchainApplication/manage/hooks/usePinBlockchainApplication';
 import DialogLink from '@theme/dialog/link';
-import { TertiaryButton } from '@theme/buttons';
-import Icon from '@theme/Icon';
-import { getLogo } from '@token/fungible/utils/helpers';
+import FlashMessageHolder from 'src/theme/flashMessage/holder';
+import { ReactComponent as CautionIcon } from '../../../../../../setup/react/assets/images/icons/caution-icon-filled.svg';
```

Comment:
Please use `@setup` alias instead

Reply:
@ikem-legend if I do this jest would scream.